BlazorLeaflet/BlazorLeaflet/LeafletInterops.cs

```diff
@@ -61,5 +89,23 @@ layer switch
 
         public static ValueTask<float> GetZoom(IJSRuntime jsRuntime, string mapId) =>
             jsRuntime.InvokeAsync<float>($"{_BaseObjectContainer}.getZoom", mapId);
+
+        private class _Bounds
+        {
+            public LatLng _southWest { get; set; }
+            public LatLng _northEast { get; set; }

```

Comment:
It would be nice to keep naming consistency, you can follow [these guidelines](https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md) if you wish, as these were used during the development of the project.

Reply:
Normally I would, but the reason I have this class is that for some reason, the components of the JSON for the bounds property are  `_southWest` and `_northEast` rather than something without underscores. There isn't any way to customize deserialization of the JavaScript objects yet. The simplest way to make this work was for me to make a private version of the `Bounds` class with property names matching the JSON, and just use it to deserialize and convert to a regular `Bounds` class.